src/main/java/uk/gov/hmcts/reform/coh/Application.java

```diff
@@ -3,14 +3,23 @@
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.Bean;
 import org.springframework.scheduling.annotation.EnableScheduling;
 import springfox.documentation.swagger2.annotations.EnableSwagger2;
 
+import java.time.Clock;
+
 @SpringBootApplication
 @EnableSwagger2
 @EnableAutoConfiguration
 @EnableScheduling
 public class Application {
+
```

Comment:
What's the use of Clock?

Reply:
It allows for injecting specified time from test-level.